web/src/main/java/org/springframework/security/web/authentication/preauth/AbstractPreAuthenticatedProcessingFilter.java

```diff
@@ -86,6 +92,7 @@
 	private boolean invalidateSessionOnPrincipalChange = true;
 	private AuthenticationSuccessHandler authenticationSuccessHandler = null;
 	private AuthenticationFailureHandler authenticationFailureHandler = null;
+	private RequestMatcher authenticationRequestMatcher = new PreAuthenticatedProcessingRequestMatcher();
```

Comment:
rename to requiresAuthenticationRequestMatcher to align with AbstractAuthenticationProcessingFilter